app/services/status_update_service.rb

```diff
@@ -12,35 +9,31 @@ def initialize(reference:, status:, completed_at:, sent_at:, user: nil)
   end
 
   def call
-    DeliveryAttempt.transaction do
+    ApplicationRecord.transaction do
       delivery_attempt.update!(
```

Comment:
Slightly out-of-scope: will the method to find the `delivery_attempt` throw a 500 if it doesn't exist?

Reply:
This is covered by https://github.com/alphagov/email-alert-api/blob/10b5f4eea4ad7f45e17d7e27793d15eaf099a4f5/app/services/status_update_service.rb#L53 which will raise a ActiveRecord::NotFound